dali/ft/filecopy.cpp

```diff
@@ -3771,7 +3774,13 @@ void FileSprayer::updateTargetProperties()
     if (distributedSource)
     {
         if (distributedSource->querySuperFile()==nullptr)
```

Comment:
not new, but what if it is a super being read?
Also not new, but it's misleading that this code to update distributedSource is in a method called "updateTargetProperties". For clarity (now in this PR), I think it should be moved outside of it.